FBSDKShareKit/FBSDKShareKit/FBSDKShareAPI.h

```diff
@@ -48,6 +49,14 @@
 @property (nonatomic, copy) NSString *graphNode;
 
 /*!
+ @abstract The access token to use instead of the global [FBSDKAccessToken currentAccessToken].  The access token must
+ have the "publish_actions" permission granted.
+ @discussion Set this value when sharing to a destination (ex. Page) that requires a separate access token from that
+ of the currently logged-in user.
+ */
+@property (nonatomic, strong) FBSDKAccessToken *accessTokenOverride;
```

Comment:
Let's just call this `accessToken` and document that it will default to `[FBSDKAccessToken currentAccessToken]`